tests/integration/csv.feature

```diff
@@ -11,6 +11,17 @@ Feature: Testing the CSV API
         And the "columns.0" property equals "title"
         Then the guzzle status code should be 200
 
+    Scenario: Uploading a CSV file with OS9 line endings
+        Given that I want to make a new "CSV"
+        And that the post file "file" is "tests/datasets/ushahidi/sample_os9.csv"
```

Comment:
this will always pass as long as the file has the content, it doesn't care that the line endings are wrong, for some reason (if I disable the feature for line endings, it still passes)

Reply:
ah.. ok I see the problem... will send an update in a few minutes